eva/executor/storage_executor.py

```diff
@@ -15,6 +15,7 @@
 from typing import Iterator
 
 from eva.catalog.catalog_type import TableType
+from eva.database import EVADB
```

Comment:
We could use `EVADatabase` instead of `EVADB` to avoid confusion with the system name